src/components/DatePickerRange.react.js

```diff
@@ -0,0 +1,306 @@
+            monthFormat={ this.props.month_format }
+            displayFormat={ this.props.display_format }
+          />
+        );
+    }
+}
+
+DatePickerRange.propTypes = {
+    id: PropTypes.string,
+
+    /**
+     * Specifies the starting date for the component, best practice is to pass
+     * value via datetime object
+     */
+    start_date: PropTypes.string,
+
+    /**
+     * Specifies the ending date for the component, best practice is to pass
+     * value via datetime object
+     */
+    end_date: PropTypes.string,
+
+    /**
+     * Specifies the lowest selectable date for the component,
+     * best practice is to pass value via datetime object
+     */
+    min_date_range: PropTypes.string,
+
+    /**
+     * Specifies the highest selectable date for the component,
+     * best practice is to pass value via datetime object
+     */
+    max_date_range: PropTypes.string,
+
+    /**
+     * Specifies the date that is initially presented when the user
+     * opens the calendar, best practice is to pass value via datetime
+     * object
+     */
+    initial_visible_month: PropTypes.string,
+
+    /**
+     * Text that will be displayed in the first input
+     * box of the date picker when no date is selected - replaces 'Start Date'
+     */
+    start_date_placeholder_text: PropTypes.string,
+
+    /**
+     * Text that will be displayed in the second input
+     * box of the date picker when no date is selected - replaces 'End Date'
+     */
+    end_date_placeholder_text: PropTypes.string,
+
+    /**
+     * If True, the calendar will automatically open when cleared
+     */
+    reopen_calendar_on_clear: PropTypes.bool,
+
+    /**
+     * Number of calendar months that are shown when calendar is opened
+     */
+    number_of_months_shown: PropTypes.number,
+
+    /**
+     * If True, calendar will open in a screen overlay portal
+     */
+    with_portal: PropTypes.bool,
+
+    /**
+     * If True, calendar will open in a full screen overlay portal, will
+     * take precedent over 'withPortal' if both are set to true
+     */
+    with_full_screen_portal: PropTypes.bool,
+
+    /**
+     * Specifies what day is the first day of the week, values must be
+     * from [0, ..., 6]
+     */
+    first_day_of_week: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6]),
+
+    /**
+     * Specifies a minimum number of nights that must be selected between
+     * the startDate and the endDate
+     */
+    minimum_nights: PropTypes.number,
+
+    /**
+     * If True the calendar will not close when the user has selected a value
+     * and will wait until the user clicks off the calendar
+     */
+    stay_open_on_select: PropTypes.bool,
+
+    /**
+     * If True the calendar will display days that rollover into
+     * the next month
+     */
+    show_outside_days: PropTypes.bool,
+
+    /**
+     * Specifies the format that the month will be displayed in the calendar,
+     * valid formats are variations of "MM YY"
```

Comment:
This is a little confusing to me. Why is `YY` in the `month_format` section? It would be helpful to have a couple of examples here, like:
```
...valid formats are variations of "MM YY". For example:
- "MM" will render as "12"
- "MMMM" will render as "1212"
```

or however it works